### PR TITLE
Remove unused packages and files

### PR DESCRIPTION
**Pull request by cesarvarela**

### CODE REVIEW

File: site/gatsby-site/package.json [hunk 1]
```diff
@@ -21,7 +21,6 @@
     "@graphql-tools/schema": "^8.3.3",
     "@graphql-tools/wrap": "^8.4.6",
     "@mdx-js/react": "^2.3.0",
-    "@philpl/buble": "^0.19.7",
     "@postlight/mercury-parser": "^2.2.1",
     "@sendgrid/mail": "^7.7.0",
```
Comment (pdcp1) [inline site/gatsby-site/package.json:25]: We should remove this `@sendgrid/mail` package as well

File: site/gatsby-site/netlify.toml [hunk 2]
```diff
@@ -10,7 +10,7 @@ TERM = "xterm"
   [plugins.inputs]
     enable = false
   [plugins.inputs.postBuild]
-    enable = true
+    enable = false
```
Comment (pdcp1) [inline site/gatsby-site/netlify.toml:13]: What is this `postBuild` step for?

**Review by pdcp1 — CHANGES_REQUESTED** [review]
Please fix conflicts

File: site/gatsby-site/netlify.toml (hunk 2 above)
Comment (cesarvarela) [reply site/gatsby-site/netlify.toml:13]: woot that's a leftover

**Comment by pdcp1** [comment]
@cesarvarela Do you have a link to do a smoke test to the whole site?

**Review by pdcp1 — APPROVED** [review]
The code looks good to me 👍 
I also did a smoke test on the whole site and it looks good as well

**Pull request merged**